model-api/bpmn-model/src/test/java/org/operaton/bpm/model/bpmn/DefinitionsTest.java

```diff
@@ -138,12 +133,7 @@ public void shouldNotAffectComments() throws IOException {
     definitions.getImports().add(importElement);
 
     // validate model
-    try {
-      Bpmn.validateModel(bpmnModelInstance);
-    }
-    catch (ModelValidationException e) {
-      Assert.fail();
-    }
+    Bpmn.validateModel(bpmnModelInstance);
```

Comment:
Does JUnit 5 fail explicitly upon thrown exceptions? This case here is handled differently from the case in line 113 and I don't quite understand why

Reply:
I have changed, because I did not see a sense on catching and failing. There is a small difference, but in the end both cases would make the test fail.

ModelValidationException is a RuntimeException. If it occurs, it would be catched by the test engine and make the test fail. That is the intention.

But yes, L133 I have overseen that and I will make it the same.

To simulate that I have made the validation fail:
![Screenshot 307](https://github.com/user-attachments/assets/f12b495f-775a-44ea-845c-16b5b809ab33)

This is how it would be handled before:
![Screenshot 308](https://github.com/user-attachments/assets/5d088367-a821-40be-9218-36a5ca3e28b6)

And this is now:
![Screenshot 309](https://github.com/user-attachments/assets/89f8207f-e2f3-4759-84ec-fc3d3c59af01)